pkgs/development/python-modules/asdf/default.nix

```
@@ -1,98 +1,73 @@
 { lib
 , asdf-standard
 , asdf-transform-schemas
-, astropy
+, attrs
 , buildPythonPackage
 , fetchFromGitHub
-, fetchpatch
-, importlib-resources
+, fsspec
+, importlib-metadata
 , jmespath
-, jsonschema
 , lz4
 , numpy
 , packaging
+, psutil
 , pytest-astropy
 , pytestCheckHook
 , pythonOlder
 , pyyaml
 , semantic-version
+, setuptools
 , setuptools-scm
 }:
 
 buildPythonPackage rec {
   pname = "asdf";
-  version = "2.13.0";
-  format = "pyproject";
+  version = "3.2.0";
+  pyproject = true;
 
   disabled = pythonOlder "3.8";
```

```suggestion
  disabled = pythonOlder "3.9";
```
https://github.com/asdf-format/asdf/blob/694494d2c77cf0609b4f4e52b8a52c686f040602/pyproject.toml#L7

---

Thanks, I'll try to not forget about this next time